addons/stock/views/product_views.xml

```diff
@@ -120,6 +166,18 @@
                         <field name="is_storable"/>
                         <field name="tracking" invisible="not is_storable" groups="stock.group_production_lot"/>
                     </div>
+                    <field name="show_qty_update_button" invisible="1"/>
+                    <label for="qty_available" class="oe_inline" invisible="type != 'consu' or not is_storable" groups="stock.group_stock_user"/>
+                    <div class="d-flex gap-2 align-items-baseline" invisible="type != 'consu' or not is_storable" groups="stock.group_stock_user">
```

Comment:
can't be is_storable if the product is not consu. Half of the condition is useless